src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        int arraySize = 6; // Можна змінити на інший розмір
+        Figure[] figures = new Figure[arraySize];
+
+        // Заповнення першої половини масиву випадковими фігурами
+        for (int i = 0; i < arraySize / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a new FigureSupplier inside the loop. In this case, it's done correctly as the FigureSupplier instance is created outside the loop.